tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -3007,7 +3095,7 @@ TEST_F(CompilerTest, CompileHlsl2015ThenFail) {
   VERIFY_ARE_EQUAL(status, E_INVALIDARG);
   VERIFY_SUCCEEDED(pResult->GetErrorBuffer(&pErrors));
   LPCSTR pErrorMsg = "HLSL Version 2015 is only supported for language services";
-  CheckOperationResultMsgs(pResult, &pErrorMsg, 1, false, false);
+  CheckOperationResultMsgs(pResult, &pErrorMsg, 1, false, false); /// use this for the failing cases
```

Comment:
Not sure why this random comment was added (it's harmless) but is a change outside the context of other changes you are making.  You could clean it up on next revision when you delete the commented out test case.